### PR TITLE
Add beakerlib framework

### PR DESCRIPTION
**Pull request by Koncpa**

### CODE REVIEW

File: Sanity/runtest.sh [hunk 1]
```diff
@@ -1198,29 +1193,32 @@ rlJournalStart
             # 10 - return
             popd
             popd
-
-        rlPhaseEnd
-    }
+        else
+            echo "DAST test is not executed, helm missing!"
```
Comment (sopos) [inline Sanity/runtest.sh:1197]: `rlLog`, `rlLogInfo`, or `rlLogWarning` might be better than just `echo`. The same below.

Comment (Koncpa) [reply Sanity/runtest.sh]: Outdated, just for debugging purposes.

File: Sanity/main.fmf [hunk 2]
```diff
@@ -4,6 +4,7 @@ contact: Martin Zelený <mzeleny@redhat.com>
 component:
   - tang
 test: ./runtest.sh
+framework: beakerlib
```
Comment (sopos) [inline Sanity/main.fmf:7]: In the common test repository, we have it in the root node and let the inheritance to work for us.
https://github.com/RedHat-SP-Security/tests/blob/master/main.fmf

Comment (Koncpa) [reply Sanity/main.fmf]: Good idea, I'll adjust it

**Review by sarroutbi — APPROVED** [review]
Changes LGTM

**Pull request merged**